core/src/main/java/bisq/core/app/BisqExecutable.java

```diff
@@ -76,11 +76,13 @@
 @Slf4j
 public abstract class BisqExecutable implements GracefulShutDownHandler, BisqSetup.BisqSetupListener {
 
+    private static final boolean isBisq = false;
+
     private final String fullName;
     private final String scriptName;
     private final String version;
 	
-	private final Preferences prefs;
+	private static Preferences prefs = Preferences.userRoot().node("Bisq");;
```

Comment:
If you want to achieve the behaviour of first time app start, that is done in a different style in many placed. I don't htink its a good ide to introduce this new style and the context of node is weird to me.

Reply:
How do you check for run?